client/server/boot/index.js

```diff
@@ -22,7 +21,6 @@ export default function setup() {
 	app.enable( 'trust proxy' );
 
 	app.use( cookieParser() );
-	app.use( userAgent.express() );
```

Comment:
What was this doing? Just setting up `request.useragent`? Anything we need to provide back compat for?

Reply:
> What was this doing? Just setting up `request.useragent`? Anything we need to provide back compat for?

Yeah, the middleware is what adds `request.useragent`. The other changes in this pull request are essentially the compatibility changes to migrate off existing use of that property.